shopfloor_reception/services/reception.py

```diff
@@ -571,15 +582,20 @@ def _set_quantity__by_product(self, picking, selected_line, product):
                 selected_line,
                 message=self.msg_store.wrong_record(product),
             )
-        selected_line.qty_done += 1
+        # if selected_line.picked:
+        #     selected_line.quantity += 1
+        # else:
+        #     selected_line.quantity = 1
+        selected_line.qty_picked += 1
+        selected_line.picked = True
```

Comment:
As suggested here, I think smth like this would be useful.
https://github.com/OCA/stock-logistics-workflow/pull/2031#discussion_r2219307240
It would take care of setting the picked value, and might raise an exception if qty after increment is higher than the demand.
```python3
def qty_picked_increment(self, qty=1):
    ...
```
Then we would just call 
```python3
try:
    ml.qty_picked_increment() # with or without a qty
```
Otherwise we will have to write the same thing over and over again.

Why not having this in `shopfloor/actions/stock...` so it could be used everywhere else?